<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CanCloseDialogClass</class>
 <widget class="QDialog" name="CanCloseDialogClass">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save changes</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Save changes to the following items?</string>
     </property>
     <property name="buddy">
      <cstring>FileList</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="FileList"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="YesButton">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="NoButton">
       <property name="text">
        <string>Don't save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>YesButton</tabstop>
  <tabstop>NoButton</tabstop>
  <tabstop>CancelButton</tabstop>
  <tabstop>FileList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
